src/transitivity/transitivity.jl

```diff
@@ -0,0 +1,30 @@
+"""
+    ```transitiveclosure!(g::SimpleGraph, selflooped = false)```
+Compute the transitive closure of an undirected graph.
+If `selflooped` is true, add self loops to the graph.
```

Comment:
Maybe we should add to the documentation, that if the graph already has self-loops, but `selflooped` is set to `false`, then there self-loops will also be removed. We are currently not doing that for the directed version, but maybe we also should.

Reply:
After thinking about it, I think that the current implementation is the right one. If one wants to modify the self loops, one should do it separately. The transitive closure is only taking care of edges _between_ different vertices, not _within_ the same vertex.

I will mention it in the docs.